### PR TITLE
Support StatsManager::readAllValue include p99

### PR DESCRIPTION
**Pull request by laura-ding**
`curl -G "http://127.0.0.1:13070/get_stats"` include p99

### CODE REVIEW

File: src/common/stats/StatsManager.cpp [hunk 1]
```diff
@@ -249,6 +249,36 @@ void StatsManager::readAllValue(folly::dynamic& vals) {
                 vals.push_back(std::move(stat));
             }
         }
+
+        // add Percentile
+        for (auto range = TimeRange::FIVE_SECONDS; range <= TimeRange::ONE_HOUR;
```
Comment (dangleptr) [inline src/common/stats/StatsManager.cpp:254]: I want to know why not add one StatsMethod kind such as P99 directly?

Comment (laura-ding) [reply src/common/stats/StatsManager.cpp:254]: `p99` does not belong to the `StatsMethod`, it belong to HistogramMethod.

Comment (dangleptr) [reply src/common/stats/StatsManager.cpp:254]: make sense. Maybe we need HistogramMethod kind such as p50, p90, p95, p99

Comment (laura-ding) [reply src/common/stats/StatsManager.cpp:254]: Good point, 2.0 can add it.

**Pull request merged**